Create ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md (#2357)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-116-fix-permissions-check-in-can-edit-dashboard-6881.md

---
date: ""

> title: fix permissions check in canEditDashboard (#6881)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix permissions check when editing dashboards
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59456
version: 1018.503.116
---

> fix permissions check in canEditDashboard (#6881)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, users were unable to edit dashboards even if they had the appropriate permissions. This was due to an incorrect permissions check in the canEditDashboard function. The permissions check has now been fixed, allowing users with the correct permissions to edit dashboards as expected. This change ensures that the dashboard editing functionality aligns with the user's assigned permissions, providing a more consistent and secure experience.
```

> Fixed dashboard delete button permissions check to use correct roles. Previously, the button was enabled for users without deletion rights, causing confusion. Now, it's only enabled for users with proper permissions, accurately reflecting access rights.

Reviewer: ```suggestion
Previously, users could delete dashboards without having the required permissions. This issue has been fixed. Now the **Delete** button is only enabled for users with the appropriate permissions, accurately reflecting the access rights.
```
